### DISCUSSION TITLE
Constraints on names?

### DISCUSSION THREAD

**Discussion by rsheeter** [fontra/fontra#1086] [General]
QQ, can a layer or axis name be _anything_ or are there limitations? For example, an axis named an emoji, all whitespace, ... is there any restriction?

**Comment by anthrotype** [comment]
they are just strings for humans to read in order to identify something, why would/should there be any particular restrictions?

**Comment by justvanrossum** [comment]
There are no theoretical restrictions, although I can’t rule out an accidental restriction due to an implementation error.

**Comment by rsheeter** [comment]
Just to confirm, there's no length limit? No "can't be exclusively white space" requirement? If so I think we've answered the question and can close.
